converters/toZigbee.js

```diff
@@ -2439,6 +2439,9 @@ const converters = {
                 } else {
                     await entity.command('closuresWindowCovering', 'stop', {}, utils.getOptions(meta.mapped, entity));
                 }
+
+                // Xiaomi curtain does not send position update on stop, request this.
+                await entity.read('genAnalogOutput', [0x0055]);
```

Comment:
Moreover, reading `genAnalogOutput` does not return the correct value immediately after stopping. To read the correct value, you need to wait about ~0.8 second.

Reply:
I think it would be better to exclude ZNCLDJ11LM? I think it was impacting more than ZNJLBL01LM because it was added before the support. In theory, it is kind of redundant for ZNJLBL01LM after I changed the code in this PR, but the removal did cause issue before, that's why I added it back. Also, thanks for the info about the ~0.8s, this explains exactly why I had the problem I was trying to fix.

Reply:
I still think that for the time being it is better to do the reading only for ZNJLBL01LM. I guess in most curtains `currentPositionLiftPercentage` returns the actual position of the stop. Moreover, I consider reading a bad decision, because it can also return the wrong value if requested too early.

P.S. I also have two ZNJLBL01LM and will be able to check the issue described in https://github.com/Koenkk/zigbee2mqtt/issues/9447#issuecomment-1109717041 over the weekend.

Reply:
> I think it was impacting more than ZNJLBL01LM because it was added before the support.

It was added by Koen in 9b7de72 for ZNCLDJ11LM original issue: #4585

Reply:
> My ZNJLBL01LM does not respond to currentPositionLiftPercentage, reading it always gets a 0

Sadly, this is Aqara E1 :)

> I think excluding is better because it will not impact the existing devices as much, it won't suddenly change behavior unless it is actually addressing the issue for that device.

Okay, then I think it's worth adding a comment why we exclude devices from reading.

Reply:
Yes, exactly. So for the 4 curtain/cover aqara/xiaomi devices, 2 are for roller shades, SRSC-M01 and the E1. I did not add this device, I came into adding functionalities by copying from existing code and modifying existing stuff, and I'd imagine that is also how many others did it. Up on checking, the E1 does not respond to `currentPositionLiftPercentage` hence `fz.xiaomi_curtain_position_tilt` is not useful at all, so I removed this in this PR. But it is sill in the other devices' definations, I don't really know if they needs it since I don't have any of them to test, and I don't know if the person added it actually understanded it, so I tried to be minimally invasive. Hope this can explain why I thought excluding is better. However, I am not a professional dev by any means, so I don't really know what I am doing. I'll add some comments for now, hope that my mind set did not add too much unnecessary code.